code/game/machinery/vending/vendor_types/crew/pilot_officer.dm

```diff
@@ -284,8 +284,12 @@ GLOBAL_LIST_INIT(cm_vending_clothing_dropship_crew_chief, list(
 		return combined
 	if(user.job == JOB_DROPSHIP_CREW_CHIEF)
 		return GLOB.cm_vending_clothing_dropship_crew_chief
+	if(user.job == JOB_OPERATIONS_CREW_CHIEF)
+		return GLOB.cm_vending_clothing_dropship_crew_chief
 	if(user.job == JOB_CAS_PILOT)
 		return GLOB.cm_vending_clothing_pilot_officer
 	if(user.job == JOB_DROPSHIP_PILOT)
 		return GLOB.cm_vending_clothing_pilot_officer
+	if(user.job == JOB_OPERATIONS_PILOT)
+		return GLOB.cm_vending_clothing_pilot_officer
```

Comment:
Would probably by simpler using a switch(user.job) but it's not strictly necessary.